pyriemann/embedding.py

```diff
@@ -0,0 +1,134 @@
+"""Embedding covariance matrices via manifold learning techniques."""
+
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+from pyriemann.utils.distance import distance
+    
+class Embedding(BaseEstimator, TransformerMixin):
+
+    """Embed SPD matrices into an Euclidean space of smaller dimension.
+
+    It uses diffusion maps to embed the SPD matrices into an Euclidean space.
+    The Euclidean distance between points in this new space approximates 
+    the Diffusion distance (also called commute distance) between vertices 
+    of a graph where each SPD matrix is a vertex. 
```

Comment:
Perfect, 
try to put it in the docstring following the [numpydoc convention](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) (see the reference section)